charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -17,6 +17,9 @@ spec:
       {{ .Values.clickhouse.backup.backup_user }}/networks/ip: "0.0.0.0/0"
       {{ template "clickhouse.backupPasswordValue" . }}
       {{- end}}
+      {{- if .Values.clickhouse.users }} 
```

Comment:
I find this a bit confusing as we have already other values (like: `clickhouse.user` defined). 

What about `additionalUsersConfig`?